<launch>

  <arg name="use_sim_time" default="false" />

  <node pkg="asb_canopy_spraying_task" exec="canopy_volume_estimation_node" name="canopy_volume_estimation">
    <remap from="points_in" to="/scan_front_multilayer/points_filtered"/>
    <param name="use_sim_time" value="$(var use_sim_time)" />
    <param name="resolution" value="0.2" />
    <param name="max_range" value="5.0" />
  </node>

  <node pkg="asb_canopy_spraying_task" exec="spraying_regulator.py" name="spraying_regulator" />

</launch>
